api/v1beta1/v1beta2_condition_consts.go

```diff
@@ -0,0 +1,262 @@
+	// the same condition type exists.
+	BootstrapConfigReadyV1beta2Condition = "BootstrapConfigReady"
+
+	// InfrastructureReadyV1beta2Condition reports if an object's infrastructure is ready.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	InfrastructureReadyV1beta2Condition = "InfrastructureReady"
+
+	// MachinesReadyV1beta2Condition surfaces detail of issues on the controlled machines, if any.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	MachinesReadyV1beta2Condition = "MachinesReady"
+
+	// MachinesUpToDateV1beta2Condition surfaces details of controlled machines not up to date, if any.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	MachinesUpToDateV1beta2Condition = "MachinesUpToDate"
+
+	// ScalingUpV1beta2Condition reports if an object is scaling up.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	ScalingUpV1beta2Condition = "ScalingUp"
+
+	// ScalingDownV1beta2Condition reports if an object is scaling down.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	ScalingDownV1beta2Condition = "ScalingDown"
+
+	// RemediatingV1beta2Condition surfaces details about ongoing remediation of the controlled machines, if any.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	RemediatingV1beta2Condition = "Remediating"
+
+	// DeletingV1beta2Condition surfaces details about progress of the object deletion workflow.
+	// Note: This condition type is defined to ensure consistent naming of conditions across objects.
+	// Please use object specific variants of this condition which provides more details for each context where
+	// the same condition type exists.
+	DeletingV1beta2Condition = "Deleting"
+
+	// PausedV1beta2Condition reports if reconciliation for an object or the cluster it belongs to is supported.
```

Comment:
Can we simply say reports if the object is paused or its reconciliation is paused? This sounds like we have objects that don't support reconciliation

Reply:
fixed